### PR TITLE
Windows Flutter build updates

### PR DESCRIPTION
**Pull request by danrmiller**
Uncomment downloading flutter windows SDK and fetch version 3.10.3 instead of 3.7.12.

### CODE REVIEW

**Review by sneurlax — APPROVED** [review]

File: scripts/windows/deps.ps1 [hunk 1]
```diff
@@ -1,3 +1,6 @@
+$FLUTTER_SDK_ZIP = "C:\development\flutter_windows_3.10.3-stable.zip"
+$FLUTTER_SDK_URL = "https://storage.googleapis.com/flutter_infra_release/releases/stable/windows/flutter_windows_3.10.3-stable.zip"
```
Comment (sneurlax) [inline scripts/windows/deps.ps1:2]: we're on 3.10.6 now but otherwise I'd use this updated script for windows dev setup

**Review by sneurlax — CHANGES_REQUESTED** [review]
since this submission, we've moved on from 3.10.3 to 3.10.6

we should test this on Windows sometime

**Comment by sneurlax** [comment]
We are now on Flutter 3.13.0.

**Comment by sneurlax** [comment]
We're now on Flutter 3.16.0

**Comment by sneurlax** [comment]
See https://github.com/cypherstack/stack_wallet/pull/800/files#diff-9d87e8f7634435500e079b2c1df3fbedf1ab06de98a395fdf1722d69e837fb58 for an update to 3.16.0

**Comment by danrmiller** [comment]
stale

**Pull request closed without merging**